CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # myuw-notifications versions
 
+## 1.3.3
+
+### Fixed
+- Accessing elements within notifications for screen reader users
```

Comment:
[Given that 1.3.2 has not yet been released](https://www.npmjs.com/package/@myuw-web-components/myuw-notifications), might this change be included in the same release with the change to automatically close the notification menu upon tabbing out of it?